### PR TITLE
pex: message ergonomics

### PR DESCRIPTION
**Pull request by marbar3778**
## Description 

Change pex message naming convention

### CODE REVIEW

**Comment by erikgrinaker** [comment]
See https://github.com/tendermint/tendermint/pull/5974 for corresponding Tendermint Go change. Went with `PexRequest` over `AddressRequest`, and also made some other changes to separate the PEX protocol from other protocols.

File: proto/p2p/pex.proto [hunk 1]
```diff
@@ -6,15 +6,15 @@ option go_package = "github.com/tendermint/tendermint/proto/tendermint/p2p";
 import "tendermint/p2p/types.proto";
 import "gogoproto/gogo.proto";
 
-message PexRequest {}
+message AddrsRequest {}
```
Comment (erikgrinaker) [inline proto/p2p/pex.proto:9]:
```suggestion
message PexRequest {}
```

File: proto/p2p/pex.proto [hunk 2]
```diff
@@ -6,15 +6,15 @@ option go_package = "github.com/tendermint/tendermint/proto/tendermint/p2p";
...
 
-message PexAddrs {
+message AddrsResponse {
```
Comment (erikgrinaker) [inline proto/p2p/pex.proto:11]:
```suggestion
message PexResponse {
```

File: proto/p2p/pex.proto [hunk 3]
```diff
@@ -6,15 +6,15 @@ option go_package = "github.com/tendermint/tendermint/proto/tendermint/p2p";
...
   repeated NetAddress addrs = 1 [(gogoproto.nullable) = false];
```
Comment (erikgrinaker) [inline proto/p2p/pex.proto:12]:
```suggestion
  repeated NetAddress addresses = 1 [(gogoproto.nullable) = false];
```

File: proto/p2p/pex.proto [hunk 4]
```diff
@@ -6,15 +6,15 @@ option go_package = "github.com/tendermint/tendermint/proto/tendermint/p2p";
...
 }
 
 message Message {
```
Comment (erikgrinaker) [inline proto/p2p/pex.proto:15]:
```suggestion
message PexMessage {
```

File: proto/p2p/pex.proto [hunk 5]
```diff
@@ -6,15 +6,15 @@ option go_package = "github.com/tendermint/tendermint/proto/tendermint/p2p";
...
   oneof sum {
-    PexRequest pex_request = 1;
-    PexAddrs   pex_addrs   = 2;
+    AddrsRequest  addrs_request  = 1;
+    AddrsResponse addrs_response = 2;
```
Comment (erikgrinaker) [inline proto/p2p/pex.proto:18]:
```suggestion
    PexRequest  pex_request  = 1;
    PexResponse pex_response = 2;
```

File: spec/p2p/messages/pex.md [hunk 6]
```diff
@@ -20,17 +20,17 @@ PexRequest is an empty message requesting a list of peers.
...
-### PexAddrs
+### PexResponse
 
-PexAddrs is an list of net addresses provided to a peer to dial.
+PexResponse is an list of net addresses provided to a peer to dial.
 
 | Name  | Type                               | Description                              | Field Number |
 |-------|------------------------------------|------------------------------------------|--------------|
-| addrs | repeated [NetAddress](#netaddress) | List of peer addresses available to dial | 1            |
+| addrs | repeated [PexAddress](#Pexaddress) | List of peer addresses available to dial | 1            |
```
Comment (erikgrinaker) [inline spec/p2p/messages/pex.md:29]:
```suggestion
| addresses | repeated [PexAddress](#Pexaddress) | List of peer addresses available to dial | 1            |
```

File: spec/p2p/messages/pex.md [hunk 7]
```diff
@@ -42,7 +42,7 @@ NetAddress provides needed information for a node to dial a peer.
...
 Message is a [`oneof` protobuf type](https://developers.google.com/protocol-buffers/docs/proto#oneof). The one of consists of two messages.
 
-| Name        | Type                      | Description                                          | Field Number |
-|-------------|---------------------------|------------------------------------------------------|--------------|
-| pex_request | [PexRequest](#pexrequest) | Empty request asking for a list of addresses to dial | 1            |
-| pex_addrs   | [PexAddrs]                | List of addresses to dial                            | 2            |
+| Name         | Type                      | Description                                          | Field Number |
+|--------------|---------------------------|------------------------------------------------------|--------------|
+| pex_request  | [PexRequest](#pexrequest) | Empty request asking for a list of addresses to dial | 1            |
+| pex_response | [PexAddrs]                | List of addresses to dial                            | 2            |
```
Comment (erikgrinaker) [inline spec/p2p/messages/pex.md:48]:
```suggestion
| pex_response | [PexResponse]                | List of addresses to dial                            | 2            |
```

Comment (marbar3778) [reply spec/p2p/messages/pex.md:48]: thanks

**Review by erikgrinaker — APPROVED** [review]

**Pull request merged**